mmv1/third_party/terraform/transport/config.go.erb

```diff
@@ -429,6 +433,39 @@ func SetEndpointDefaults(d *schema.ResourceData) error {
 	return nil
 }
 
+// baseTransport returns the base HTTP transport. It starts with the
+// default transport and makes some tweaks to match best practices
+// from google-api-go-client, as well as ensuring that IPv6 does
+// not get used in environments that don't support it.
+func baseTransport() (http.RoundTripper, error) {
+	trans := http.DefaultTransport.(*http.Transport).Clone()
+	// Increase MaxIdleConnsPerHost due to reported performance issues under load in the
+	// GCS client.
+	trans.MaxIdleConnsPerHost = 100
```

Comment:
If I'm understanding correctly, these tweaks are from [here](https://github.com/googleapis/google-api-go-client/blob/707b9b287dc37fb07c2553c0391f83995f1420a6/transport/http/dial.go#L174)?

Reply:
Great, you did mention it in the comment, but I'll leave the link here just for posterity